checkov/terraform/runner.py

```diff
@@ -631,5 +631,5 @@ def get_graph_resource_entity_config(self, entity, entity_context):
         definition_path = entity_context.get('definition_path', [])
         entity_config = entity['config_']
         for path in definition_path:
-            entity_config = entity_config[path]
+            entity_config = entity_config.get(path, entity_config)
```

Comment:
@arielkru i think we need to understand why this key doesnt exist. if its not related to secrets ill take this

Reply:
also curious about it, seems more like a bug